scripts/pages/home/view.mjs

```diff
@@ -0,0 +1,631 @@
+    openNavButton: document.getElementById('openbtn'),
+    profileAnchor: document.getElementById('profile'),
+    matchesAnchor: document.getElementById('matches'),
+    matchesSortButtonsArray: document.getElementsByClassName('matches-table__sort-button'),
+    heroesSortButtonsArray: document.getElementsByClassName('heroes-table__sort-button'),
+    heroesAnchor: document.getElementById('heroes'),
+    statisticsAnchor: document.getElementById('statistics'),
+    matchIdCopyArray: document.getElementsByClassName('text-copy'),
+
+    matchesLastPressedIndex: null,
+    heroesLastPressedIndex: null,
+
+    renderPlaceholders() {
+        for (let i = 0; i < 25; i++) {
+            this.matchesTable.append(this.matchesTemplate.content.cloneNode(true));
+            i < 8 ? this.heroesTable.append(this.heroesTemplate.content.cloneNode(true)) : null;
+        }
+    },
+
+    openNav() {
+        let navbar = document.querySelector('.navbar');
+        navbar.classList.toggle('active');
+        console.log("ok");
+    },
+
+    scrollToTop() {
+        document.body.scrollTop = 0;
+        document.documentElement.scrollTop = 0;
+    },
+
+    toTop(buttonToTop, buttonToTopMark) {
+        if (window.scrollY > 300) {
+            buttonToTop.classList.add('show');
+        } else {
+            buttonToTop.classList.remove('show');
+            buttonToTopMark.classList.remove('show');
+        }
+    },
+
+    copyIdToClipboard(button) {
+        let temp = document.createElement('textarea');
+        temp.style.position = "fixed";
+        temp.style.opcity = "0";
+        temp.textContent = button.getAttribute("data-label");
+        console.log(button.getAttribute("data-label"));
+
+        document.body.appendChild(temp);
+        temp.select();
+        document.execCommand('copy');
+        document.body.removeChild(temp);
+    },
+
+    pressButtonMatches(buttonIndex) {
+        console.log(buttonIndex, this.matchesLastPressedIndex);
+
+        this.matchesSortButtonsArray[buttonIndex].classList.add('pressed');
+
+        if (this.matchesLastPressedIndex !== null) {
+            this.matchesSortButtonsArray[this.matchesLastPressedIndex].classList.remove('pressed');
+        }
+
+        this.matchesLastPressedIndex = buttonIndex;
+
+        console.log(buttonIndex, this.matchesLastPressedIndex);
+    },
+
+    pressButtonHeroes(buttonIndex) {
+        console.log(buttonIndex, this.heroesLastPressedIndex);
+
+        this.heroesSortButtonsArray[buttonIndex].classList.add('pressed');
+
+        if (this.heroesLastPressedIndex !== null) {
+            this.heroesSortButtonsArray[this.heroesLastPressedIndex].classList.remove('pressed');
+        }
+
+        this.heroesLastPressedIndex = buttonIndex;
+
+        console.log(buttonIndex, this.heroesLastPressedIndex);
+    },
+
+    renderProfile(data) {
+        const profileInfo = document.getElementsByClassName('profile-info')[0];
+
+        profileInfo.innerHTML = `
```

Comment:
This is a whole new approach here. It looks inconsistent with the one you are using below by creating DOM nodes. So you'll have to pick one of them. If you decided to choose "string approach", I would recommend keeping those innerHTML literals somewhere outside of the `view` object.  Maybe, it worth creating a separate file for them